### ISSUE TITLE
Set item always in last position?

### ISSUE THREAD

**Issue by huytd** [ManifestWebDesign/angular-gridster#377]
In short, I want to make the dashboard which has the `Add Widget` button that will always be in the last position. Is this possible with current Angular Gridster version?

**Comment by Sheepzez** [comment]
+1 - I'm also interested in having newly added widgets appear at the top, rather than the bottom.

**Comment by puvaasam** [comment]
I'm also looking for the solution..
Requirement: Newly added widget should comes first on grid..

**Comment by Sheepzez** [comment]
I actually found a solution to this. Setting the x and y position parameters to 0,0 will always put it at the top left of gridster.

**Comment by puvaasam** [comment]
Yeah, you are correct. but in our case all the other widgets should push to one step back based on newly added widget size

_Current Dashboard structure_
 row1: W-0-1  W-0-2  W-0-3
 row2: W-1-1 W-1-2  W-1-3
 row3: W-2-1 W-2-2  W-2-3

For example, if i add a new widget (name : w-new1) into the dashboard. the newly add one(w-new1) should appears first and other should moved one step back.

_expected format:_
 row1: w-new1 W-0-1  W-0-2  
 row2: W-0-3 W-1-1 W-1-2  
 row3: W-1-3 W-2-1 W-2-2  
 row4: W-2-3

**Comment by glmaster123** [comment]
+1

**Comment by shnigi** [comment]
+1

**Comment by Mersal-Mohamed2** [comment]
is there a solution
